cmd/ocm/token/cmd.go

```diff
@@ -111,10 +123,21 @@ func run(cmd *cobra.Command, argv []string) error {
 		return fmt.Errorf("Can't create connection: %v", err)
 	}
 
-	// Get the tokens:
-	accessToken, refreshToken, err := connection.Tokens()
-	if err != nil {
-		return fmt.Errorf("Can't get token: %v", err)
+	var accessToken string
+	var refreshToken string
+
+	if args.generate {
+		// Get new tokens:
+		accessToken, refreshToken, err = connection.Tokens(15 * time.Minute)
```

Comment:
would it make sense to make this configurable?
```sh
ocm token --generate # 15 minutes
ocm token --generate=10 # 10 minutes
# etc...
```
maybe not, but wondering, since it's being hard-coded

Reply:
There is an upper limit so I didn't want to allow the user to supply nonsensical values like say 60 minutes

Reply:
makes sense